src/main/webapp/zoom.html

```diff
@@ -5,6 +5,7 @@
   <title>Seniors Connect</title>
   <link rel="stylesheet" href="styles/techstyle.css">
```

Comment:
Is techstyle.css necessary for this page? What is the scope of techstyle.css?

Reply:
No its not, removed.